identity/handler.go

```diff
@@ -524,3 +590,70 @@ func (h *Handler) delete(w http.ResponseWriter, r *http.Request, ps httprouter.P
 
 	w.WriteHeader(http.StatusNoContent)
 }
+
+// fandom start - allow admins to remove multifactor authentication for an identity
+
+// swagger:parameters adminDeleteIdentityCredential
+// nolint:deadcode,unused
+type adminDeleteIdentityCredential struct {
+	// ID is the identity's ID.
+	//
+	// required: true
+	// in: path
+	ID string `json:"id"`
+	// credentialType is the type of credential
+	//
+	// required: true
+	// in: path
+	CredentialType string `json:"credentialType"`
+}
+
+// swagger:route DELETE /identities/{id}/credential/{credentialType} v0alpha2 adminDeleteIdentityCredential
+//
+// # Update an Identity
+//
+// Allow admins to remove multifactor authentication for an identity
+//
+//	Consumes:
+//	- application/json
+//
+//	Produces:
+//	- application/json
+//
+//	Schemes: http, https
+//
+//	Security:
+//	  oryAccessToken:
+//
+//	Responses:
+//	  204: emptyResponse
+//	  404: jsonError
+//	  500: jsonError
+func (h *Handler) deleteMultifactorCredential(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
```

Comment:
Issue: This functionality is not migrated

Reply:
See https://github.com/Wikia/kratos/pull/97/commits/909636d21142ee4a4dfeb2f5c92dfb1cea2611ed

Reply:
It was replaced by the upstream implementation.
https://fandom.atlassian.net/browse/PLATFORM-9038